libr/core/cconfig.c

```diff
@@ -2936,7 +2936,7 @@ R_API int r_core_config_init(RCore *core) {
 #endif
 #if __x86_64__
 	SETI ("dbg.glibc.ma_offset", 0x000000, "Main_arena offset from his symbol");
-	SETI ("dbg.glibc.fc_offset", 0x00240, "First chunk offset from brk_start");
+	SETI ("dbg.glibc.fc_offset", 0x00280, "First chunk offset from brk_start");
```

Comment:
I think this depends from the Glibc version. Ideally, it should be set in runtime, not in the compilation time.

Reply:
- The changes I made find the fc_offset during runtime and does not depend on dbg.glibc.fc_offset